ceph.spec.in

```diff
@@ -584,7 +585,9 @@ mv $RPM_BUILD_ROOT/sbin/mount.fuse.ceph $RPM_BUILD_ROOT/usr/sbin/mount.fuse.ceph
 
 #set up placeholder directories
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/ceph
+%if ! 0%{?_with_systemd}
```

Comment:
I would prefer you added a sysVinit variable to the spec file. 

(This is some thing I have been thinking of doing we will see what ken thinks of this idea.)